### PR TITLE
Fix and improve recommended config

### LINKED ISSUE #109: "recommended" config is unusable
See https://github.com/amilajack/eslint-plugin-compat/pull/29#issuecomment-372733878

### PR DESCRIPTION
**Pull request by tbroyer**
Fixes #109

### CODE REVIEW

**Comment by amilajack** [comment]
I think it would be nice if you mentioned the recommended config in the readme

**Comment by tbroyer** [comment]
> I think it would be nice if you mentioned the recommended config in the readme

Ah yes, good catch! Done.

I also added `"env": { "browser": true }` to the config.

**Comment by sobolevn** [comment]
Any chance to get this merged please?

File: src/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import requireIndex from 'requireindex'; // eslint-disable-line flowtype-errors/
 import recommended from './config/recommended';
 
 
-export const config = {
+export const configs = {
```
[the remark this answers is not in the document]
Comment (tbroyer) [reply src/index.js]: Afaict, with `config` you're not exposing/exporting any config: https://eslint.org/docs/developer-guide/working-with-plugins#configs-in-plugins

[the remark this answers is not in the document]
Comment (tbroyer) [reply src/index.js]: See also https://github.com/amilajack/eslint-plugin-compat/pull/29#issuecomment-372733878

[the remark this answers is not in the document]
Comment (ljharb) [reply src/index.js]: Sure, but someone might have been requiring it directly.

To be safe, this object would be exported under both names.

**Comment by amilajack** [comment]
Thanks for this!

**Comment by amilajack** [comment]
@ljharb thanks for reviewing

**Comment by sobolevn** [comment]
@amilajack could you please release this fix? It is a blocker for me: https://github.com/wemake-services/wemake-vue-template/issues/601

**Pull request merged**